riak/tests/test_riak.py

```diff
@@ -30,9 +30,12 @@ def test_check(aggregator, check, instance):
 
 
 @pytest.mark.usefixtures('dd_environment')
-def test_bad_config(aggregator, check):
+def test_bad_config(aggregator, instance):
+    instance.update({"url": "http://localhost:5985"})
+    check = Riak('riak', {}, [instance])
+
     with pytest.raises(socket.error):
-        check.check({"url": "http://localhost:5985"})
+        check.check(instance)
 
-    sc_tags = ['url:http://localhost:5985']
+    sc_tags = ['my_tag', 'url:http://localhost:5985']
```

Comment:
I don't understand why `my_tag` was not submitted before. Do you have more details?

Reply:
Because before the test was no using the same fixture. It was just using `{"url": "http://localhost:5985"}` while now its using common.INSTANCE with an updated url